modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadLaws.kt

```diff
@@ -82,10 +87,11 @@ object MonadLaws {
         val b = just(a + 1).bind()
         val c = just(b + 1).bind()
         c
-      }.run(this)
+      }.run(M)
```

Comment:
same as above, there's no need to have a local capture for documentation purposes when calling by name does the job